Xamarin.PropertyEditing.Mac/Controls/PropertyEditorControl.cs

```diff
@@ -157,5 +165,18 @@ internal new TViewModel ViewModel
 			get { return (TViewModel)base.ViewModel; }
 			set { base.ViewModel = value; }
 		}
+
+		public override bool IsDynamicallySized => true;
+
+		public override nint GetHeight (EditorViewModel vm)
+		{
+			nint baseHeight = base.GetHeight (vm);
+			var realVm = (PropertyViewModel)vm;
+			if (realVm != null && realVm.IsVariant) {
+				return baseHeight * 2;
```

Comment:
Why is it multiplied? This is just going to lead to uneven sizes when trying to adjust the multiplier for different base sizes. Just add the fixed size of the variant header.

Reply:
@ermau Could you clarify what you mean by "add the fixed size of the variant header"?

Reply:
The variant header (as in the list of variations for this variant) is a fixed or calculate-able size, just add that to the base height plus margins.